### ISSUE TITLE
Using Ctrl a in place Ctrl b in "tmux"

### ISSUE THREAD

**Issue by triantares** [Elive/elive-for-servers#6]
I noticed that "tmux" in the elive version uses <Ctrl a> where most tutorials on the web use <Ctrl b>.
I've always used "screen" until now, so I'm not unhappy with that combo :) but ....
Shouldn't that be mentioned somewhere in the README.md ?

**Comment by TheTechRobo** [comment]
That was already the config on Elive, but yeah it should be added to the README or at least when you do press Ctrl b it says "Elive uses CTRL+A ;)" or smth

**Comment by Thanatermesis** [comment]
> Shouldn't that be mentioned somewhere in the README.md ?

Yes, the tmux and zsh are the most important "elivized" things from this tool, very good and important to the users know these features and possibilities, you* are welcome to read the conf files and fill up the README file with the features info & details :)

The website of Elive itself also will need this info, but that's a future thing (a full menu section called Features with different pages showing features in details, one of them of course will be the shell, so tmux / zsh / terminology / etc)

**Comment by triantares** [comment]
Actually, I like the solution that @TheTechRobo just gave. That way we'll have "caught" all those that didn't fully read the docs as well.
It would even be a nice addition for the elive tmux instance too. So maybe a better solution to change that instance overall.

**Comment by Thanatermesis** [comment]
Hum, not exactly possible, ctrl+b is used for something :), more exactly: when you are in a tmux inside a tmux (yes, sometimes this is needed, but I don't remember an example now), so you need to control the second tmux instead of trigger the main one

**Comment by triantares** [comment]
Hmm, looking at .tmux.conf I see you've done "unbind C-b", so shouldn't that make it available to echo the comment below it?
```
###########################
#  Configuration
###########################
# tmux prefix (main hotkey trigger) changed from ^b to ^a
set -g prefix C-a
unbind C-b
bind-key C-a send-prefix
# Important: Hotkey ^b is remaped to ^a, just like in "screen" app

```
Albeit "remaped" typo should then be corrected to "remapped" :smile:

**Comment by Thanatermesis** [comment]
> Albeit "remaped" typo should then be corrected to "remapped" 

meh, its just a comment typo

> but maybe that mapping isn't a good idea, it's kinda confusing anyway. thinking

not really, if you use "a" it will affect tmux, if you open a tmux inside a tmux (unset TMUX first) you need to use "b", so "a" affects the first one (let's say switching or creating windows, or like: ctrl + a ; page-up to scroll up)

**Comment by triantares** [comment]
Yeah, that's clear to me now but not to anyone working with existing howtos/tutorials from the web.
I don't know, until now I've always solely used "screen" but what would the send-prefix in a standard nested tmux session?
The same as in non-nested one?

Anyway, when not changing to that trick, it might be another option to state that clearly in the welcome screen.

I know, I know, I found out fast enough but I also know to go looking for diffs in standard usage of CML stuff on Elive.... so I wasn't surprised. :rofl:

**Comment by Thanatermesis** [comment]
ok, all fixed, now tmux shows a friendly help, and some compact improvements and easenesseses have been added :)

> apug
> elive-skel upgrade .tmux.conf
> tmux

**Comment by triantares** [comment]
Yeah, saw that change.
That's good as the initial welcome screen moves out of view after a while. 
Might be an idea to add those shortcuts in the README.md here as well....considering you can't type "help" if there's already a long process running.
